plugin/src/main/resources/jenkins/plugins/openstack/compute/slaveopts/LauncherFactory/KAFKA/config.jelly

```diff
@@ -0,0 +1,9 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+    <f:section title="KAFKA remote agent launch">
+        <f:entry>
+            <p>Jenkins will wait for the Kafka connection to be initiated from the slave itself. Note this plugin does not initiate
```

Comment:
I know this was inspired by JNLP launcher, but you get me to see this is actually in